### ISSUE TITLE
Grafana v6 beta 1 / worldping 1.2.4 : constantly requires a datasource update

### ISSUE THREAD

**Issue by NotSqrt** [raintank/worldping-app#215]
Hi,

I recently installed the worldping plugin on my hosted grafana (version 5.2.x at that time) and I upgraded grafana to v6 beta 1 today.

Now I have the red message on all worldping pages:
```
worldPing now supports Grafana Alerting!
To start using alerting, an update to your worldPing data source is required.
```
And clicking on the `Update now` button gives 2 green alerts `Datasource updated`, but the red message keeps coming back.

Thanks !

**Comment by NotSqrt** [comment]
More generally, it seems the plugin needs some updates for grafana 6 : the new endpoint form does not seem to be submittable.

**Comment by thomasjungblut** [comment]
this is biting us as well, do you have any ETA for it to be fixed?

**Comment by daniellee** [comment]
> Now I have the red message on all worldping pages:

Looks like an incorrect version check:

![image](https://user-images.githubusercontent.com/434655/53001829-d7c8a600-342b-11e9-9c4a-7b913fb4c8cd.png)

We made a change to the version check code in Grafana which is probably causing this.

> More generally, it seems the plugin needs some updates for grafana 6 : the new endpoint form does not seem to be submittable.

This seems to work for me. Could you provide more details?

**Comment by NotSqrt** [comment]
CSS partially broken on, checkboxes :
![selection_081](https://user-images.githubusercontent.com/1997005/53004966-8374f480-3432-11e9-8219-c67d92e9d5b1.png)

I see errors when trying to leave an unsaved form by clicking on any link of the page (404 error and uncaught promise error), and can't leave the form without clicking on "Cancel" or hard refresh the page.

I haven't retried to create an endpoint, but the form seems to work, so that's good.

Thanks !

**Comment by nayanashettyft** [comment]
Hi,

I am try to install the worldping plugin on my hosted grafana (version v6.0.0 (34a9a62)). This error looks related to what I am [seeing.](url)

I see a red message when trying to configure world ping with the APIKey :

"Unexpected error" and the network tools shows an "Unauthorized" quotas call.
<img width="1197" alt="screen shot 2019-03-02 at 22 53 38" src="https://user-images.githubusercontent.com/25954099/53688596-1882c200-3d3e-11e9-9128-dc22a66c0f34.png">

Thanks !

**Comment by jostrasser** [comment]
I have a question to the message...
 "worldPing now supports Grafana Alerting!
To start using alerting, an update to your worldPing data source is required."

Will the original way to create alerts via the "Endpoint Config" settings page be deprecated or removed in a future update of the worldping plugin?

Thanks JO!

**Comment by Paul-Reed** [comment]
@mattttt It would be great if this could be fixed, so that we can use alerts with WorldPing.

**Comment by thomasjungblut** [comment]
any idea when we can expect a fix for this?

**Comment by Paul-Reed** [comment]
@alexanderzobnin I'm on worldping 1.2.4 and for some reason I cannot update it!!
```
pi@raspberrypi:~ $ grafana-cli plugins update raintank-worldping-app
Error: ✗ Could not find dist/plugin.json or plugin.json on  raintank-worldping-app in /var/lib/grafana/plugins
```
If I check in my `/var/lib/grafana/plugins` and `/var/lib/grafana/plugins/raintank-worldping-app` there is no `plugin.json`.

If I try and update the plugin using Grafanas GUI, it doesn't update either.

**Comment by jostrasser** [comment]
Don’t know why this issue is closed, it still exists...

**Comment by DanCech** [comment]
Please try the 1.2.5 release, it was fixed in #221

**Comment by jostrasser** [comment]
Thanks!
